frameworks/elastic/tests/config.py

```diff
@@ -241,7 +244,10 @@ def get_document(index_name, index_type, doc_id, service_name=PACKAGE_NAME):
 
 
 def _curl_api(service_name, method, role="master"):
-    host = "http://" + sdk_hosts.autoip_host(service_name, "{}-0-node".format(role), _master_zero_http_port(service_name))
+    host = 'http://'
```

Comment:
A more pythony way to do this would be:
```python
host = 'https://' if ELASTIC_TLS_ENABLED else 'http://'
```
We could also consider using `urlib` to construct the URL:
```python
>>> import urllib.parse
>>> urllib.parse.urljoin('/media/path/', 'js/foo.js')
'/media/path/js/foo.js'
```